adapter/internal/oasparser/model/swagger_test.go

```diff
@@ -44,7 +43,7 @@ func TestSetInfoSwaggerWebSocket(t *testing.T) {
 	err = json.Unmarshal(apiJsn, &apiYaml)
 	assert.Nil(t, err, "Error occured while parsing api.yaml")
 	var mgwSwagger MgwSwagger
-	err = mgwSwagger.PopulateSwaggerFromAPIYaml(apiYaml, constants.WS)
+	err = mgwSwagger.PopulateSwaggerFromAPIYaml(apiYaml)
```

Comment:
its `mgwSwagger.` so we know its doing somthing on mgwSwagger. so we can simply call this `populateFromAPIYaml`. It seems we use `populate`, `get`, `build`, `generate`, `process` terms interchangeably everywhere. we should pick correct one based on the task inside the function.

Reply:
+1 for the suggestion to pick the right term based on the task done within the function. Will keep a note on that here onwards. Renaming to `PopulateFromAPIYaml`